package/MDAnalysis/lib/util.py

```diff
@@ -212,6 +214,182 @@ def filename(name, ext=None, keep=False):
     return name if isstream(name) else str(name)
 
 
+def get_reader_for(filename, format=None):
```

Comment:
`lib/util` is getting a little long, but it's nice having these functions in `lib` so you can import in `core.Universe` cleanly